examples/favorites_advanced/bloc_favorites_advanced/lib/main.dart

```diff
@@ -14,8 +15,14 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) => MultiBlocProvider(
         providers: [
-          BlocProvider<NavigationBarBloc>(
-            create: (context) => NavigationBarBloc(),
+          // BlocProvider<NavigationBarBloc>(
+          //   create: (context) => NavigationBarBloc(),
+          // ),
+          BlocProvider<PuppyListBloc>(
```

Comment:
@StanislavPrime, is there a reason to not use home_providers.dart ?

Reply:
I am using the the method from home_providers.dart in the home_page.dart file and I moved the BlocProvider<PuppyListBloc> to be also there